### PR TITLE
Add a test for TF mixed precision

### PR DESCRIPTION
**Pull request by jplu**
# What does this PR do?

This PR adds a test to check if our TF models are float16 compliant or not. It also helps me to detect what are those that have to be fixed.

### CODE REVIEW

**Comment by jplu** [comment]
I don't think that the tests that fails are related to this PR.

**Review by sgugger — APPROVED** [review]
LGTM! That's a good thing to tackle as we really want all our TF models to work in mixed precision!

**Review by LysandreJik — APPROVED** [review]
LGTM! Nice objective!

File: tests/test_modeling_tf_longformer.py [hunk 1]
```diff
@@ -340,14 +340,25 @@ def test_for_multiple_choice(self):
 
     @slow
     def test_saved_model_with_attentions_output(self):
-        # longformer has special attentions which are not
-        # compatible in graph mode
+        # This test don't pass because of the error:
```
Comment (patrickvonplaten) [inline tests/test_modeling_tf_longformer.py:343]: nice!

**Review by patrickvonplaten — APPROVED** [review]
Can we re-run the ci tests to make that they are unrelated? There seemed to be related to network issues

**Pull request merged**